pytorch_lightning/trainer/trainer.py

```diff
@@ -885,7 +885,7 @@ def fit(
 
             # train
             if self.tpu_id is not None:
-                self.tpu_train(self.tpu_id, model)
+                self.tpu_train(model)
             else:
                 xmp.spawn(self.tpu_train, args=(model,), nprocs=self.tpu_cores, start_method=start_method)
```

Comment:
as you can see in this line, the tpu_train function is passed to this spawn function and it expect the tpu_core_idx.
It's not unused afterall.

Reply:
dam... we need to get TPU test asap...
cc: @luiscape